flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ExecutionContext.java

```diff
@@ -219,6 +246,10 @@ public ClusterID getClusterId() {
 		return catalogs;
 	}
 
+	public SessionState getSessionState() {
```

Comment:
get nullable field should return `Optional`

Reply:
This member field is not nullable, would just remove the `@Nullable` tag.